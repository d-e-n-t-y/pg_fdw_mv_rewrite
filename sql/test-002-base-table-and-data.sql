create table test (key text, hidden text, value text);

insert into test (select 'key1', 'hidden1', generate_series (1, 10));

insert into test (select 'key2', 'hidden2', generate_series (1, 1e6));
