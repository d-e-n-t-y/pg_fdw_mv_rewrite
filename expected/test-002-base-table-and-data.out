create table test (key text, hidden text, value text);
CREATE FOREIGN TABLE test_lb (key text, hidden text, value text) server loopback OPTIONS (schema_name 'public', table_name 'test');
insert into test (select 'key1', 'hidden1', generate_series (1, 10));
insert into test (select 'key2', 'hidden2', generate_series (1, 1e6));
